packages/rx_bloc_generator/pubspec.yaml

```diff
@@ -10,19 +10,19 @@ environment:
   sdk: ">=3.0.5 <4.0.0"
 
 dependencies:
-  analyzer: ^6.0.0
+  analyzer: ">=6.0.0 <8.0.0"
   build: ^2.4.0
   code_builder: ^4.10.0
   collection: ^1.17.1
-  dart_style: ">=2.3.1 <3.0.0"
+  dart_style: ^3.0.0 
```

Comment:
Same concern as above. There might be packages that are still not migrated to the version 3.0.0 so to avoid any conflicts we can use the following constraint: `dart_style: >=2.3.1 <4.0.0`

Reply:
Done